packages/scaffold-ui/src/views/w3m-swap-view/index.ts

```diff
@@ -185,21 +185,24 @@ export class W3mSwapView extends LitElement {
     )
     const amount = target === 'toToken' ? this.toTokenAmount : this.sourceTokenAmount
     const price = target === 'toToken' ? this.toTokenPriceInUSD : this.sourceTokenPriceInUSD
-    let value = parseFloat(amount) * price
+    const marketValue = NumberUtil.parseLocalStringToNumber(amount) * price
 
-    if (target === 'toToken') {
-      value -= this.gasPriceInUSD || 0
-    }
+    // Why do we need to subtract the gas price here?
+    /*
+     * If (target === 'toToken') {
+     *   marketValue -= this.gasPriceInUSD || 0
+     * }
+     */
 
     return html`<w3m-swap-input
       .value=${target === 'toToken' ? this.toTokenAmount : this.sourceTokenAmount}
-      ?disabled=${this.loadingQuote && target === 'toToken'}
+      .disabled=${target === 'toToken'}
```

Comment:
why remove the loading ?

Reply:
that input should be always disabled, unless we want to add some logic to calculate the swap when changing the toToken input. Doesn't work right now

Reply:
Does disabling it changes it's styles like making some elements less visible? We should make sure it's not doing it